tests/scene/test_person.py

```
@@ -198,7 +199,7 @@ def test_updateAll_layer_props_init():
         ]
     }
     scene = Scene()
-    assert scene.read(data) == None
+    assert scene.read(DATA) == None
```

Ruff (0.13.3)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)